src/domains/wallet-app/api/payment-wallet/v1/_get_wallets_by_user.tpl

```diff
@@ -0,0 +1,79 @@
+<policies>
+    <inbound>
+    <base />
+    <set-variable name="walletToken"  value="@(context.Request.Headers.GetValueOrDefault("Authorization", "").Replace("Bearer ",""))"  />
+    <!-- Get User IO START-->
+    <send-request ignore-error="true" timeout="10" response-variable-name="user-auth-body" mode="new">
+        <set-url>@("${io_backend_base_path}/pagopa/api/v1/user?version=20200114")</set-url> 
+        <set-method>GET</set-method>
+        <set-header name="Accept" exists-action="override">
+          <value>application/json</value>
+        </set-header>
+        <set-header name="Authorization" exists-action="override">
+          <value>@("Bearer " + (string)context.Variables.GetValueOrDefault("walletToken"))</value>
+        </set-header>
+    </send-request>
+    <choose>
+      <when condition="@(((IResponse)context.Variables["user-auth-body"]).StatusCode != 200)">
+        <return-response>
+          <set-status code="502" reason="Bad Gateway" />
```

Comment:
it is correct to return `502` in this case, but this status code is not present in the `openapi`, can you add it?